hypothesis-python/src/hypothesis/core.py

```diff
@@ -1159,24 +1150,21 @@ def template_condition(data):
             list(runner.interesting_examples.values())[0].buffer)
         with BuildContext(data):
             return data.draw(search)
-    if (
-        runner.valid_examples <= settings.min_satisfying_examples and
+    if runner.valid_examples == 0 and (
         runner.exit_reason != ExitReason.finished
     ):
         if settings.timeout > 0 and run_time > settings.timeout:
-            raise Timeout((
+            raise Timeout((  # pragma: no cover
```

Comment:
Why is this no cover now?

Reply:
Something about `min_satisfying_examples` going away interacted badly with timeouts in `find()`.  

I tried several variations on the condition above and on the test itself to no effect; and ended up just deleting it in 416504ad25077467219b8116ce4850ccb633a3fa.

Reply:
The test wasn't working with the change to `min_satisfying_examples`, and with timeout going away so soon I eventually just gave up on working out why.